packaging_automation/update_pgxn.py

```diff
@@ -64,4 +64,5 @@ def update_pgxn_files(project_version: str, template_path: str, exec_path: str):
     if not args.is_test:
         run(f'git push --set-upstream origin {pr_branch}')
         create_pr(github_token, pr_branch, commit_message, REPO_OWNER, PROJECT_NAME, main_branch)
+    if not args.pipeline and not args.is_test:
```

Comment:
To prevent the error below I added this check
https://github.com/citusdata/packaging/runs/3609157981?check_suite_focus=true